src/components/tools/text-toolbar.tsx

```diff
@@ -1,10 +1,14 @@
-import React from "react";
+import React, { useEffect } from "react";
 import ReactDOM from "react-dom";
 import { Editor, EFormat, handleToggleSuperSubscript } from "@concord-consortium/slate-editor";
 import { IFloatingToolbarProps, useFloatingToolbarLocation } from "./hooks/use-floating-toolbar-location";
 import { useSettingFromStores } from "../../hooks/use-stores";
 import { TextToolbarButton } from "./text-toolbar-button";
 import { IRegisterToolApiProps } from "./tool-tile";
+// Note this isn't listed as a direct dependency, it is transitive from the
+// slate-editor. Perhaps it should be exported by the slate-editor
+import EventEmitter from "eventemitter3";
```

Comment:
I agree that it would make sense for it to be exported by the slate-editor library. Some recommended sets of lint rules will flag cases of importing from indirect dependencies.